crates/ast_node/Cargo.toml

```diff
@@ -13,6 +13,7 @@ bench = false
 proc-macro = true
 
 [dependencies]
+bytecheck = "0.6.8"
```

Comment:
`ast_node` is proc macro, so I don't think we need this

Reply:
👍